controller/konnect/ops/ops_kongpluginbinding.go

```diff
@@ -241,7 +253,14 @@ func kongPluginWithTargetsToKongPluginInput(
 			pluginInput.Service = &sdkkonnectcomp.PluginService{
 				ID: lo.ToPtr(t.GetKonnectStatus().ID),
 			}
-		// TODO(mlavacca): add support for KongRoute
+		case *configurationv1alpha1.KongRoute:
+			id := t.GetKonnectID()
+			if id == "" {
+				return nil, fmt.Errorf("KongRoute %s is not configured in Konnect yet", client.ObjectKeyFromObject(t))
+			}
+			pluginInput.Route = &sdkkonnectcomp.PluginRoute{
+				ID: lo.ToPtr(t.GetKonnectStatus().ID),
```

Comment:
```suggestion
				ID: lo.ToPtr(id),
```
Same could be applied to `KongService` code above.

Reply:
0d231de3880e7cd3ed4382df0513c8261ea5b26f